packages/app/src/components/nominationInfo/healthProviderSearch/index.js

```diff
@@ -26,35 +27,63 @@ const SearchHealthProvider = () => {
   }, [NominationsData]);
 
   return (
-    <>
-      <section className="search-result-card">
-        <table className="search-result-table">
-          <thead>
-            <tr>
-              <th>Health Provider Search Results</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr className={styles.bold}>
-              <td>Application Name</td>
-              <td>Provider's Name</td>
-              <td>Patient Name</td>
-              <td>Recieved Date </td>
-            </tr>
-            {SearchHealthcareProvider?.map((result) => (
-              <tr key={result.id}>
-                <td>
-                  <Link to={`/nomination/${result.id}`}>{result.nominationName}</Link>
-                </td>
-                <td>{result.providerName}</td>
-                <td>{result.patientName}</td>
-                <td>{result.dateReceived}</td>
-              </tr>
-            ))}
-          </tbody>
-        </table>
-      </section>
-    </>
+    <table className="new-files-table">
+      <thead>
+        <tr>
+          <td className="add-padding-left new-files-title">
+            <FontAwesomeIcon icon="file-image" color="green" />
+            <h1>Health Provider Search Results</h1>
+          </td>
+          <td></td>
+          <td></td>
+          <td></td>
+        </tr>
+      </thead>
+      <tbody>
+        <tr className="home-new-files-headers">
+          <td className="add-padding-left width-column">
+            {' '}
+            <h2 className="sortable-column">
+              <strong>Application Name </strong>
+            </h2>
+          </td>
+          <td className="width-column">
+            <h2 className="sortable-column">
+              <strong> HP Name </strong>
+            </h2>
+          </td>
+          <td className="width-column">
+            {' '}
+            <h2 className="sortable-column">
+              <strong>Patient Name </strong>
+            </h2>
+          </td>
+          <td className="width-column">
+            {' '}
+            <h2 className="sortable-column">
+              <strong>Recieved Date </strong>
+            </h2>
+          </td>
+          <td className="last-column">
+            <h2 className="sortable-column">
+              <strong> </strong>
```

Comment:
Should there be something in this tag?

Reply:
Updated the branch to include both comments. Removed the extra strong tags and white space.